### PR TITLE
FE libs v6.1.0

### LINKED ISSUE #617: unescapeHTML doesn't actually unescape HTML entities properly
### Describe your bug

The `unescapeHTML` helper doesn't actually unescape HTML entities properly, resulting in them showing up in post selectors, etc.

### Steps to Reproduce

Run `unescapeHTML` on this string `Mia: &#8220;Moj &#8216;guilty pleasure&#8217; je plagijarizam&#8221;`

### Expected behavior

`unescapeHTML` should return `Mia: "Moj 'guilty pleasure' je plagijarizam"`.

Current behaviour is that the original string is returned.

### Screenshots, screen recording, code snippet

_No response_

### Environment info

_No response_

### Please confirm that you have searched existing issues in this repo.

Yes

### Please confirm that you have searched in our documentation and couldn't find the answer.

Yes

### Please confirm that your bug occurs with all plugins uninstalled and with the default WordPress theme active.

No

### LINKED ISSUE #618: Wrong documentation for Responsive component
### Describe your bug

The readme for the `Responsive` component provides documentation for `ColorPalleteCustom`, instead of `Responsive`.

See https://github.com/infinum/eightshift-frontend-libs/tree/develop/scripts/components/responsive

### Steps to Reproduce

https://github.com/infinum/eightshift-frontend-libs/tree/develop/scripts/components/responsive

### Expected behavior

Expected documentation for Responsive.

### Screenshots, screen recording, code snippet

_No response_

### Environment info

_No response_

### Please confirm that you have searched existing issues in this repo.

No

### Please confirm that you have searched in our documentation and couldn't find the answer.

No

### Please confirm that your bug occurs with all plugins uninstalled and with the default WordPress theme active.

No

### PR DESCRIPTION
**Pull request by goranalkovic-infinum**
# Description

Fixes:
- refactored `unescapeHTML` to work way more reliably (should handle everything now!) - closes #617 
- fixed readme and doc comment for `Responsive` - closes #618
- `Collapsable`, `CollapsableComponentUseToggle` and `CompactResponsive` are now smarter with outline indentation and nesting

New features:
- additional/custom classes can be passed to more components now (`CollapsableComponentUseToggle`, `SimpleHorizontalSingleSelect`, `CustomSlider`, `CustomRangeSlider` )
- added new utility classes for display, aligns, flexbox, ...
- added new sizes for utility classes
- refactored `InlineNotification` to make it fit in better with current components
- refactored `LinkEditComponent` to make it fit in better with current components
- added an opt-in class to allow animating the `inherit` arrow
- a couple of default ES admin colors were changed to have more contrast and look better
- `CustomSlider` and `CustomRangeSlider` have better value displays now - the width is auto-calculated based on the `max` value to prevent the layout from moving when changing for example from 9 to 10
- `IconLabel` has a subtitle gap option now
- added new icons

# Screenshots / Videos

New `InlineNotification`
image

New `LinkEditComponent`
image
image
image

Smarter collapsable component outline indentation
image

# Linked documentation PR

\-

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,27 @@ All notable changes to this project will be documented in this file.
 
 This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).
 
+## [6.0.6] - 2022-05-17
```
Comment (dingo-d) [inline CHANGELOG.md:7]: The changed list alone makes this a `6.1.0` version...

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@eightshift/frontend-libs",
-	"version": "6.0.5",
+	"version": "6.0.6",
```
Comment (dingo-d) [inline package.json:3]:
```suggestion
	"version": "6.1.0",
```

File: scripts/components/advanced-color-picker/advanced-color-picker.js [hunk 3]
```diff
@@ -118,6 +121,7 @@ export const AdvancedColorPicker = (props) => {
 
 	return (
 		<BaseControl
+			className={additionalClasses ?? ''}
```
Comment (dingo-d) [inline scripts/components/advanced-color-picker/advanced-color-picker.js:124]: Just a note for the future that these kinds of changes are the reason for the minor version update.

Semver is: `Major.Minor.Security`, and adding any kind of feature, no matter how small (until it's a BC break), goes in the minor release.

**Review by dingo-d — DISMISSED** [review]
Looks good (after the version change 😄), great job 👍🏼

**Comment by dingo-d** [comment]
@goranalkovic-infinum just check why phpstan check is failing

**Comment by goranalkovic-infinum** [comment]
> @goranalkovic-infinum just check why phpstan check is failing

@dingo-d I've seen this a couple of times before, might be tests-related?
image

**Comment by dingo-d** [comment]
@goranalkovic-infinum can you add https://github.com/infinum/eightshift-libs-stubs to the composer as a dev dependency? The `phpstan.neon.dist` is referencing them as a dependency, and they are not present as one.

**Comment by goranalkovic-infinum** [comment]
Converted to draft until stubs issue that's failing PHPStan check is solved

**Review by iobrado — DISMISSED** [review]

**Review by mbmjertan — APPROVED** [review]
Apart from the phpstan / stubs issues, looks great! 😄

**Review by dingo-d — APPROVED** [review]

**Review by mbmjertan — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]

**Pull request merged**